internal/test_helpers/course_definition.yml

```
@@ -112,6 +112,17 @@ extensions:
       [rpush-command]: https://redis.io/docs/latest/commands/rpush/
       [lrange-command]: https://redis.io/docs/latest/commands/lrange/
 
+  - slug: "pub-sub"
+    name: "Pub-Sub"
```

```suggestion
    name: "Pub/Sub"
```

(Consistent with Redis's official docs)